suites/covid19_vci/vc_headers.rb

```diff
@@ -0,0 +1,21 @@
+module Covid19VCI
+  class VCHeaders < Inferno::Test
+    title 'Verifiable Credentials contain the correct headers'
```

Comment:
This guidance is specific to Health Cards and not Verifiable Credentials more broadly. It would make sense to rename the suite accordingly too I think to be more about Health Cards instead of VCs.

```suggestion
    title 'Health Cards contain the correct headers'
```